### PR TITLE
feat(hatchery,api,ui): show worker model register console log

### PR DESCRIPTION
**Pull request by fsamin**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: sdk/error.go [hunk 1]
```diff
@@ -452,6 +454,7 @@ var errorsFrench = map[int]string{
 	ErrWorkflowConditionBadOperator.ID:           "Opérateur de condition de lancement incorrect",
 	ErrColorBadFormat.ID:                         "Format de la couleur incorrect. Vous devez utiliser le format hexadécimal (exemple: #FFFF)",
 	ErrInvalidHookConfiguration.ID:               "Configuration de hook invalide",
+	ErrWorkerModelDeploymentFailed.ID:            "Echec de déploiement du modèle de worker",
```
Comment (bnjjj) [inline sdk/error.go]:
```suggestion
	ErrWorkerModelDeploymentFailed.ID:            "Échec de déploiement du modèle de worker",
```

File: ui/src/app/views/settings/worker-model/edit/worker-model.edit.html [hunk 2]
```diff
@@ -85,6 +85,10 @@ <h2 *ngIf="workerModel && workerModel.id">{{ 'worker_model_edit_title' | transla
                       <div class="item" *ngIf="workerModel.type === 'openstack'">{{'worker_model_help_error_openstack_2' | translate}}</div>
                     </div>
                   </div>
+                    <h4>{{ 'worker_model_error_log' | translate }}</h4>
```
Comment (bnjjj) [inline ui/src/app/views/settings/worker-model/edit/worker-model.edit.html]:
```suggestion
                    <h4 *ngIf="workerModel.last_spawn_err_log">{{ 'worker_model_error_log' | translate }}</h4>
```

File: engine/api/worker_model.go [hunk 3]
```diff
@@ -146,33 +146,33 @@ func (api *API) bookWorkerModelHandler() service.Handler {
...
+		if err != nil {
+			return err
 		}
 
-		tx, errBegin := api.mustDB().Begin()
-		if errBegin != nil {
-			return sdk.WrapError(errBegin, "spawnErrorWorkerModelHandler> Cannot start transaction")
+		tx, err := api.mustDB().Begin()
+		if err != nil {
+			return err
```
Comment (richardlt) [inline engine/api/worker_model.go]: This should at least be return like sdk.WithStack(err), same for other errors never wrapped.

Comment (richardlt) [reply engine/api/worker_model.go]:
```suggestion
			return sdk.WithStack(err)
```

**Pull request merged**